starter/slack-spring-boot/src/main/kotlin/com/kreait/slack/broker/store/user/UserChangedEventReceiver.kt

```diff
@@ -0,0 +1,25 @@
+package com.kreait.slack.broker.store.user
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.kreait.slack.api.contract.jackson.SlackEvent
+import com.kreait.slack.api.contract.jackson.common.types.Member
+import com.kreait.slack.broker.receiver.EventReceiver
+import com.kreait.slack.broker.store.team.Team
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.HttpHeaders
+
+
+class UserChangedEventReceiver @Autowired constructor(private val userStore: UserStore) : EventReceiver {
+
+    override fun supportsEvent(slackEvent: SlackEvent): Boolean {
+        return slackEvent.event["type"] == "user_change"
```

Comment:
I think its a good idea to actually map all event strings into an static class/object @BenDenger can you write a story for that